flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/e2e/MySqlE2eITCase.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.e2e;
+
+import com.ververica.cdc.connectors.e2e.utils.FlinkContainerTestEnvironment;
+import com.ververica.cdc.connectors.e2e.utils.JdbcProxy;
+import com.ververica.cdc.connectors.e2e.utils.TestUtils;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+
+/** End-to-end tests for mysql-cdc connector uber jar. */
+public class MySqlE2eITCase extends FlinkContainerTestEnvironment {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MySqlE2eITCase.class);
+
+    private static final Path mysqlCdcJar = TestUtils.getResource("mysql-cdc-connector.jar");
+    private static final Path jdbcJar = TestUtils.getResource("jdbc-connector.jar");
+
+    @Test
+    public void testMySqlCDC() throws Exception {
+        List<String> sqlLines =
+                Arrays.asList(
+                        "CREATE TABLE products_source (",
+                        " `id` INT NOT NULL,",
+                        " name STRING,",
+                        " description STRING,",
+                        " weight DECIMAL(10,3),",
+                        " enum_c STRING,",
+                        " json_c STRING,",
+                        " point_c STRING,",
+                        " primary key (`id`) not enforced",
+                        ") WITH (",
+                        " 'connector' = 'mysql-cdc',",
+                        " 'hostname' = '" + INTER_CONTAINER_MYSQL_ALIAS + "',",
+                        " 'port' = '3306',",
+                        " 'username' = '" + MYSQL_TEST_USER + "',",
+                        " 'password' = '" + MYSQL_TEST_PASSWORD + "',",
+                        " 'database-name' = '" + mysqlInventoryDatabase.getDatabaseName() + "',",
+                        " 'table-name' = 'products_source',",
+                        " 'server-id' = '5800-5900',",
```

Comment:
the `server-id` range is to large here

Reply:
Is there any problem to use a large range?

Reply:
I mean this large range looks like we need 100 server ids in e2e test, but actually 4 server ids is enough

Reply:
Yes. But it should still works even we set a large range, right? I think many users will set a large range, because they may change parallelism frequently.